index.html

```diff
@@ -0,0 +1,154 @@
+        </nav>
+      </div>
+      <div class="row introduction">
+        
+        <p>"Welcome to Healthline Nutrition!"</p>
+        <h1 class="secondHeader">
+          The Definitive Guide<br />
+          to Healthy Eating in Real Life
+        </h1>
+
+        <p id="second-p">
+          healthy eating doesn’t have to be complicated. It’s entirely possible
+          to nourish your body while enjoying the foods you love.
+        </p>
+        <img src="assets/images/Health-Nutrition.jpg" alt="Nutrition" />
+        
+      </div>
+      <div class="row mainProgram" id="program">
+        <h3>Main Program</h3>
+        <hr />
+        <div class="col-sm card">
+          <img src="assets/images/vegtables.jpg" alt="Vegtables" />
+          <h4 class="card-title">Vegetables</h4>
+          <p class="card-text">
+            Add variety to grilled or steamed vegetables with an herb such as
+            rosemary..
+          </p>
+        </div>
+        <div class="col-sm card">
+          <img src="assets/images/fruits.jpg" alt=" fruits" />
+          <h4 class="card-title">Fruits</h4>
+          <p class="card-text">
+            Fresh, frozen, or canned fruits are great choices. Try fruits beyond apples and bananas such as mango, pineapple or kiwi fruit
+          </p>
+        </div>
+        <div class="col-sm card">
+          <img
+            src="assets/images/calcium.jpg"
+            alt=" calcuim-rich food image"
+          />
+          <h4 class="card-title">calcium-Rich food</h4>
+          <p class="card-text">
+            In addition to fat-free and low-fat milk, consider low-fat and fat-free yogurts without added sugars.
+          </p>
+        </div>
+        <div class="col-sm card">
+          <img src="assets/images/meat.jpg" alt=" meat picture" />
+          <h4 class="card-title">Meat</h4>
+          <p class="card-text">
+            If your favorite recipe calls for frying fish or breaded chicken, try healthier variations by baking or grilling. 
+          </p>
+        </div>
+
+        <p class="text-white" id="wholeProgram">SEE THE WHOLE PROGRAM</p>
+      </div>
+      <div class="row">
+      <section class="sports">
+        <h2>Sports</h2>
+        <hr>
+      </section>
+      
+      </div>
+      <div class="row">
+        <section class="partner">
+            <h3>Parnter</h3>
+            <hr>
+            <span class="mozilla">
+             mozilla
+            </span>
+            <span class=" google">
+              Google
+            </span>
+            <span class="naver">
+              Naver
+            </span>
+            <span  class="daumkakao">
+              daumkakao
+            </span>
+            <span  class="airbnb">
+              airbnb
+            </span>
+        </section>
+      </div>
+      <div class="row">
+        <footer>
+             <p>
+              some rights reversed 	&copy;
+            </p>
+            
+          </footer>
+      </div>
+    </div>
```

Comment:
- [ ] The footer section is missing, please add it as shown in the template:

<img width="919" alt="Screen Shot 2022-09-12 at 10 35 45 AM" src="https://user-images.githubusercontent.com/83717285/189702430-77a8dab8-429e-494c-adf6-c2cd3b74278f.png">

Reply:
Add right footer